charts/argocd-apps/ci/applicationsets-multiple-sources-values.yaml

```diff
@@ -22,3 +22,16 @@ applicationsets:
       destination:
         server: '{{url}}'
         namespace: default
+  templatePatch: |
```

Comment:
It's from https://argo-cd.readthedocs.io/en/stable/operator-manual/applicationset/Template/#template-patch